internal/watcher/watcher.go

```diff
@@ -4,8 +4,7 @@
 
 package watcher
 
-// #cgo LDFLAGS: -lwatcher -lstdc++
-// #cgo CFLAGS: -Wall -Werror
+// #cgo LDFLAGS: -lwatcher-c -lstdc++ -Wl,-rpath /usr/local/lib
```

Comment:
Will this require the watcher to be installed to `/usr/local/lib` and/or does it affect static builds?

Reply:
This can be override using `CGO_LDGLAGS`, but we'll likely be able to remove this soon: https://github.com/e-dant/watcher/issues/59